x/ibc/module.go

```diff
@@ -65,7 +65,7 @@ func (AppModuleBasic) ValidateGenesis(cdc codec.JSONMarshaler, config client.TxE
 func (AppModuleBasic) RegisterRESTRoutes(client.Context, *mux.Router) {}
 
 // RegisterGRPCRoutes registers the gRPC Gateway routes for the ibc module.
-func (a AppModuleBasic) RegisterGRPCRoutes(_ client.Context, _ *runtime.ServeMux) {
+func (AppModuleBasic) RegisterGRPCRoutes(_ client.Context, _ *runtime.ServeMux) {
```

Comment:
how come the gRPC for IBC isn't being registered?

Reply:
are we missing something? IBC should be good to go for the gRPC Gateway

Reply:
We should register gRPC for ibc as well. I think it is my mistake while reverting few changes in this PR. I'll add it back.